### ISSUE TITLE
docker-compose example

### ISSUE THREAD

**Issue by bndw** [appbaseio/dejaVu#288]
I've run the included [docker-compose.yml](https://github.com/appbaseio/dejavu/blob/dev/docker-compose.yml) but end up with what appears to be a connection error to elasticsearch:

<img width="1155" alt="image" src="https://user-images.githubusercontent.com/4248167/55442972-b4823200-5565-11e9-9648-1becfd1d90a3.png">

I've tried the following (after manually creating an index):
- using `http://elasticsearch`, `http://elasticsearch:9200`, `http://localhost:9200`, `http://127.0.0.1:9200`
- Specifying all of the above origins to the elasticsearch CORS env var, as well as just setting `*`
- Passing the environment variables explicitly to elasticsearch through the `environment` in docker-compose

Any help getting this running would be appreciated.

**Comment by siddharthlatest** [comment]
@bndw The docker-compose file whitelists localhost:1358 for CORS but you are using 127.0.0.1:1358. You can either use localhost:1358 and then the connection should work or add 127.0.0.1:1358 origin to the CORS settings. Hope this helps!

**Comment by bndw** [comment]
@siddharthlatest I've changed the origins but am still unable to connect.

Here's my `docker-compose.yml`

```yaml
version: '3'
services:
  # search engine
  elasticsearch:
    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.5.4
    container_name: elasticsearch
    ports:
      - "9200:9200"
    entrypoint:
      - elasticsearch
    environment:
      - http.port=9200
      - http.cors.enabled=true
      - http.cors.allow-origin=http://localhost:1358,http://127.0.0.1:1358
      - http.cors.allow-headers=X-Requested-With,X-Auth-Token,Content-Type,Content-Length,Authorization
      - http.cors.allow-credentials=true
    user: elasticsearch
    ports:
      - "9200:9200"
  # elasticsearch browser
  dejavu:
    image: appbaseio/dejavu:3.0.0-alpha
    container_name: dejavu
    ports:
     - "1358:1358"
    links:
      - elasticsearch
```

I manually created index `test` with curl
```
curl -X PUT "localhost:9200/test"
{"acknowledged":true,"shards_acknowledged":true,"index":"test"}
```

Here's what I get:

<img width="1151" alt="image" src="https://user-images.githubusercontent.com/4248167/55502409-586ceb80-5601-11e9-85e1-95a52417e218.png">

**Comment by siddharthlatest** [comment]
Does `curl http://elasticsearch:9200` work? I have also updated the docker-compose file. Can you try with that? It uses the latest Dejavu and ES releases.

Afaict, that's the issue. ES is accessible at something like `http://localhost:9200`, the `http://elasticsearch:9200` would only work from within the container.

**Comment by bndw** [comment]
`http://elasticsearch:9200` would definitely only work from within the Dejavu container since it's using a Docker link. I'm not sure how Dejavu uses that URL under the hood, but on the host machine ES is accessible through `http://localhost:9200`.

I've pulled your latest `docker-compose.yml` and using the following steps I was able to get this going:
1. Run the compose file with `docker-compose up`
2. Create an index called "test" in ES with `curl -X PUT "localhost:9200/test"`
3. Navigate to the Dejavu webapp at http://localhost:1358
4. Use `http://localhost:9200` as the Cluster URL
5. Use `test` as the Appname aka Index

Thanks for your help. Let me know if you'd like me to submit a PR with these steps to help other users.
